tests/annotation/bandSpec.js

```diff
@@ -21,7 +21,7 @@ describe('band', function() {
             .call(band);
 
         var path = element.children[0].children[0];
-        expect(path.getAttribute('d')).toEqual('M0,0h20v20h-20z');
+        expect(path.getAttribute('d')).toEqual('M0,0h20v20h-20Z');
```

Comment:
I looked up the difference between `Z` and `z`, MDN says:

> Apparently there is no difference between the uppercase and lowercase command.

I love the lack of certainty!